index.js

```diff
@@ -1,109 +1,145 @@
 #!/usr/bin/env node
 
-import fs from 'fs';
-import path from 'path';
+import 'babel-polyfill';
+import program from 'commander';
+import { stat, readFile, writeFile, unlink } from 'fs-extra';
```

Comment:
We can use promisified fs methods from `fs-extra`, so there is no need to use `util` and promisify them manually.